TagsCloudVisualization/TagsCloudVisualization_Test/CoordinatesConverter_Test.cs

```diff
@@ -0,0 +1,46 @@
+﻿using FluentAssertions;
+using FluentAssertions.Execution;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TagsCloudVisualization;
+
+namespace TagsCloudVisualization_Test
+{
+    class CoordinatesConverter_Test
+    {
+        [TestCase(5, 3, 5.83, 0.54)]
+        [TestCase(5, -3, 5.83, -0.54)]
+        [TestCase(-5, 3, 5.83, 2.6)]
+        [TestCase(-5, -3, 5.83, -2.6)]
+        [TestCase(50, 30, 58.3, 0.54)]
+        public void ToPolar_ShouldBeCorrect(int x, int y, double expectedRho, double expectedPhi)
+        {
+            var polar = CoordinatesConverter.ToPolar(new Point(x, y));
```

Comment:
В таких местах можно деконструктор использовать

```suggestion
            var (rho, phi) = CoordinatesConverter.ToPolar(new Point(x, y));
```